app.css

```diff
@@ -1,6 +1,319 @@
-*{
+* {
   box-sizing: border-box;
 }
-body{
+
+body {
+  margin: 0;
+  font-family: "Raleway", sans-serif;
+}
+
+ul {
+  padding-left: 0;
+  list-style: none;
+}
+
+img {
+  max-width: 100%;
+}
+
+.container {
+  margin: 0 auto;
+  max-width: 65rem;
+}
+
+.hero {
+  padding: 1rem 2rem;
+  background: url('images/surf.jpeg') no-repeat center center fixed;
+  background-size: cover;
+
+}
+
+.top-nav {
+  overflow: auto;
+  color: black;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  flex-wrap: wrap;
+}
+
+.top-nav a {
+  color: inherit;
+  text-decoration: none;
+}
+
+.top-nav img {
+  border-radius: 50%;
+  height: 2em;
+  vertical-align: middle;
+}
+
+.nav_title {
+  display: block;
+  text-align: center;
+}
+
+.hero__text h1 {
+  font-size: 4rem;
+  color: white;
+  margin-bottom: 0.1rem;
+}
+
+.hero__text p {
+  padding-top: 0.1rem;
+  margin-top: 0.1rem;
+}
+
+.top-nav__logo {
+  margin: 0;
+}
+
+.top-nav__links {
+  display: flex;
+  margin: 0 -0.5rem;
+}
+
+.top-nav__item {
+  margin: 0 0.5rem;
+}
+
+.hero__text {
+  padding: 8rem 0;
+}
+
+.article {
+  padding: 1rem;
+}
+
+.main {
+  display: flex;
+  justify-content: center;
+  flex-wrap: wrap;
```

Comment:
To `.main` add a `margin: 0 -1rem` so that the main grid content lines up with the container edges.